benchmarks/dynamo/pr_time_benchmarks/benchmark_base.py

```diff
@@ -64,29 +75,42 @@ def description(self):
         return ""
 
     @abstractmethod
-    def prepare(self):
+    def _prepare(self):
         pass
 
     @abstractmethod
-    def work(self):
+    def _work(self):
         pass
 
-    def prepare_once(self):  # noqa: B027
+    def _prepare_once(self):  # noqa: B027
         pass
 
-    def count_instructions(self):
+    def _count_instructions(self):
         print(f"collecting instruction count for {self.name()}")
-        self.prepare_once()
-
         results = []
         for i in range(10):
-            self.prepare()
+            self._prepare()
             id = i_counter.start()
-            self.work()
+            self._work()
             count = i_counter.end(id)
             print(f"instruction count for iteration {i} is {count}")
-            if i != 0:
-                results.append(count)
+            results.append(count)
+        return min(results)
+
+    def _count_compile_time_instructions(self):
+        print(f"collecting compile time instruction count for {self.name()}")
+        config.record_compile_time_instruction_count = True
```

Comment:
Add a quick comment here noting that because inner_compile (or whatever) calls record() this causes it to actually record the inner_compile ticks and not the whole thing.